src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static FileWriterService fileWriterService;
+    private static final String VALUE_FILE_PATH = "src/main/resources/outputFile.csv";

```

Comment:
The same, it is better to create link in test or beforeClass because we will change this link depend on test case